engine/lib/user_settings.php

```diff
@@ -85,6 +85,10 @@ function _elgg_set_user_name() {
 	$name = strip_tags(get_input('name'));
 	$user_guid = get_input('guid');
 
+	if (!isset($name)) {
+		return;
```

Comment:
Disregard, these are called from a hook so null is fine, but should we change the function signatures from "bool" to "bool|null" or "mixed"?